cmake/FindJeMalloc.cmake

```diff
@@ -23,6 +23,10 @@ find_path(JEMALLOC_INCLUDE_DIR jemalloc/jemalloc.h
           PATHS ${PC_JEMALLOC_INCLUDEDIR} ${PC_JEMALLOC_INCLUDE_DIRS}
           ${LIMIT_SEARCH})
 
+if(WIN32)
+  set(JEMALLOC_USE_STATIC ON)
```

Comment:
We don't use jemalloc at all, on Windows.

Reply:
If mingw64 with jemalloc installed, Cmake will found itand do link. So it not always not use.

Reply:
But we should _never_ use jemalloc on Windows. It serves no purpose. So `if(WIN32)` should instead disable jemalloc.

Reply:
After think, I hold point that as a way of performance optimization, we should enable on windows if found jemalloc already exist.

Reply:
No, jemalloc should never be used on Windows. The Windows allocator does a better job than jemalloc.

Reply:
ok